code/datums/gamemodes/battle_royale.dm

```diff
@@ -358,7 +358,23 @@ proc/hide_weapons_everywhere(var/total_battlers = 1)
 	weapon_supplies.Add(/obj/item/storage/grenade_pouch/incendiary)
 	weapon_supplies.Add(/obj/item/storage/grenade_pouch/mixed_explosive)
 	weapon_supplies.Add(/obj/item/storage/beartrap_pouch)
-
+	// Ranch Eggs
+	weapon_supplies.Add(/obj/item/reagent_containers/food/snacks/ingredient/egg/chicken/cockatrice)
```

Comment:
Currently these are replacing weapons which will exacerbate the issue of not finding a weapon early and getting steamrolled. Maybe better as a "utility" or bonus category that rolls sometimes in addition to the regular ones?